.github/workflows/ansible-tests-pr.yml

```diff
@@ -0,0 +1,74 @@
+on: 
+  pull_request_target:
+    paths:
+    - config
+    - equinix_metal
+    - templates
+    - metal_openapi.fixed.yaml
+    - oas3.stitched
+    - oas3.fetched
+    - .github/**
+  workflow_dispatch:
+
+permissions:
+  pull-requests: read
+  contents: read
+
+jobs:
+  authorize:
+    environment:
+      ${{ github.event_name == 'pull_request_target' &&
+      github.event.pull_request.head.repo.full_name != github.repository &&
+      'external' || 'internal' }}
+    runs-on: ubuntu-latest
+    steps:
+      - run: true
+
+  integration-test-pr:
+    needs: authorize
+    runs-on: ubuntu-latest
+    defaults:
+      run:
+        working-directory: .ansible/collections/ansible_collections/equinix/cloud
+    steps:
+      - name: checkout Ansible collection
+        uses: actions/checkout@v3
+        with:
+          name: equinix-labs/ansible-collection-equinix
+          path: .ansible/collections/ansible_collections/equinix/cloud
+          ref: ${{ github.event.pull_request.head.sha || github.ref }}
+
+      - name: update packages
+        run: sudo apt-get update -y
+
+      - name: install make
+        run: sudo apt-get install -y build-essential
+
+      - name: setup python 3
+        uses: actions/setup-python@v4
+        with:
+          python-version: '3.8'
+
+      - name: install dependencies
+        run: pip3 install -r requirements-dev.txt -r requirements.txt
+
+      - name: install collection
+        run: make install
+
+      - name: replace existing keys
+        run: rm -rf ~/.ansible/test && mkdir -p ~/.ansible/test && ssh-keygen -m PEM -q -t rsa -N '' -f ~/.ansible/test/id_rsa
+
+      - name: checkout Python SDK, this PR
+        uses: actions/checkout@v3
+        with:
+          path: /tmp/metal-python
+          ref: ${{ github.event.pull_request.head.sha || github.ref }}
+
+      - name: install cloned Python SDK
+        run: python -m pip install /tmp/metal-python/equinix_metal
+
+      - name: run tests
+        run: make testall
+        env:
+          METAL_API_TOKEN: ${{ secrets.METAL_API_TOKEN }}
```

Comment:
`METAL_AUTH_TOKEN`?
https://github.com/search?q=repo%3Aequinix-labs%2Fmetal-python%20METAL_AUTH_TOKEN&type=code

Reply:
Looks like the Ansible collection supports both but uses METAL_API_TOKEN for testing purposes.

https://github.com/search?q=repo%3Aequinix-labs%2Fansible-collection-equinix%20METAL_API_TOKEN&type=code

We've been using METAL_AUTH_TOKEN for consistency. For example `eval $(metal env --export)`

Reply:
`METAL_AUTH_TOKEN` and `METAL_API_TOKEN` have been added as repository secrets.